cpu/Makefile.include.gnu

```diff
@@ -9,6 +9,16 @@ endif
 export AS = $(PREFIX)as
 export LINK = $(PREFIX)gcc
 export SIZE = $(PREFIX)size
+ifeq (,$(OBJCOPY))
```

Comment:
@cgundogan, you're right. [`command -v`](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/command.html) is POSIX, `which` is not.